### ISSUE TITLE
Tokenizing issue when a SCRIPT contains a TITLE end

### ISSUE THREAD

**Issue by BenoitZugmeyer** [fb55/htmlparser2#745]
As [reported in `eslint-plugin-html`](https://github.com/BenoitZugmeyer/eslint-plugin-html/issues/141), using `</title>` in a `<script>` tag yields strange results. Ex:

```
<script>'</title>'</script>

Expected:
  opentagname script
  opentag script {}
  text '</title>'
  closetag script

Actual:
  opentagname script
  opentag script {}
  text ''
  closetag script
```

```
<script>'</title><foo>'</script>

Expected:
  opentagname script
  opentag script {}
  text '</title><foo>'
  closetag script

Actual:
  opentagname script
  opentag script {}
  text '
  opentagname foo
  opentag foo {}
  text '
  closetag foo
  closetag script
```

**Comment by fb55** [comment]
Whoa, very interesting issue. Thanks for the report!

**Comment by fb55** [comment]
Okay, fixed this and published a new version as `6.0.1`.

**Comment by BenoitZugmeyer** [comment]
Great! That was quick ☺️ thank you very much.
